code/game/objects/structures/displaycase.dm

```diff
@@ -28,17 +53,34 @@
 				src.healthcheck()
 
 
+/obj/structure/displaycase/proc/open_close(mob/user)
+	open = !open
+	alarm = !alarm
+	visible_message("<span class='notice'>The display case is now [src.open ? "opened" : "closed"].</span>")
+	update_icon()
+
 /obj/structure/displaycase/bullet_act(var/obj/item/projectile/Proj)
 	health -= Proj.get_structure_damage()
 	..()
 	src.healthcheck()
 	return
 
+/obj/structure/displaycase/proc/tripalarm(case_security)
+	var/area/case_area = get_area(src)
+	switch(case_security)
+		if(1)
+			visible_message("<span class='notice'>[src] pings cheerfully.</span>", "<span class='notice'>You hear a ping.</span>")
+		if(2)
+			priority_announcement.Announce("Warning, a company asset has been removed from it's storage case. Crew are advised to keep assets in there assigned storage cases.")
```

Comment:
```suggestion
			priority_announcement.Announce("A company asset has been removed from its storage case.")
```
This sentence flows really weirdly. It's not really crew that should be concerned with precious assets, so they shouldn't really be told by the announcement. It should be more like an unspecific warning or an alarm sound that's played.